server/src/routes/api/v1/usersRouter.js

```diff
@@ -1,19 +1,29 @@
 import express from "express";
 import passport from "passport";
 import { User } from "../../../models/index.js";
+import objection from "objection"
+const { ValidationError } = objection
+
+import cleanUserInput from "../../../services/cleanUserInput.js"
 
 const usersRouter = new express.Router();
 
 usersRouter.post("/", async (req, res) => {
-  const { email, password, passwordConfirmation } = req.body;
+  const { body } = req;
+  const formInput = cleanUserInput(req.body)
```

Comment:
On line 12 we already parse `req.body` to just `body`, so we should pass that variable to the `cleanUserInput` function